server/axios_api.js

```diff
@@ -1,61 +1,31 @@
-var axios = require("axios").default;
-var HOST = "movies-tvshows-data-imdb.p.rapidapi.com";
-var API_KEY = "030611f594msh73349b7b00af6e8p16d57ejsncbf1b45a6165";
-var URL = "https://movies-tvshows-data-imdb.p.rapidapi.com/";
-
-
-
-function get_details(id) {
-  var options = {
-    method: 'GET',
-    url: URL,
-    params: {type: 'get-movie-details', imdb: id},
-    headers: {
-      'x-rapidapi-host': HOST,
-      'x-rapidapi-key': API_KEY
-    }
-  };
-  axios.request(options).then(function (response) {
-    console.log("rating:"+response.data["imdb_rating"]+"\n"+"popularity:"+response.data["popularity"]+"\n"+"vote:"+response.data["vote_count"]);
-  }).catch(function (error) {
-    console.error(error);
-  });
-
+const { default: axios } = require("axios");
+const RATINGS = {
```

Comment:
I think ratings object shall be part of the function itself as every title will have to call it and we dont want to return same instance in case for a movie fetch fails.

Can we move below code to getDetails method

const ratings = {
  imdb: <Some value>,
  rottenTomato: <some value>
}

Also, we can avoid using all caps as variable names.